student.rb

```diff
@@ -0,0 +1,17 @@
+require_relative 'person'
+
+class Student < Person
+  def initialize(age, classroom, name = 'Unknown', parent_permission: true)
+    super(age, name, parent_permission: parent_permission)
+    @classroom = classroom
+  end
+
+  def play_hooky
+    '¯\(ツ)/¯'
+  end
+end
+
+student1 = Student.new(15, 2)
+p 'CLASS STUDENT'
+p student1.can_use_services?
+p student1.play_hooky
```

Comment:
- [ ] [Optional][highly recommended] Same here, It would be better professionally if you added all test cases for all modules in a separate file.